modules/javafx.controls/src/main/java/javafx/scene/control/skin/TabPaneSkin.java

```diff
@@ -1989,6 +1991,7 @@ private void removeReorderListeners(Node n) {
     }
 
     private ListChangeListener childListener = new ListChangeListener<Node>() {
+        @Override
```

Comment:
Please revert.

Reply:
Reverted the @Overrides inserted by eclipse.